test/e2e-go/e2e_go_suite_test.go

```diff
@@ -146,17 +146,16 @@ var _ = BeforeSuite(func(done Done) {
 	Expect(err).NotTo(HaveOccurred())
 
 	if isRunningOnKind() {
-		By("loading the project image into Kind cluster")
-		err = tc.LoadImageToKindCluster()
-		Expect(err).NotTo(HaveOccurred())
+		By("loading the required images into Kind cluster")
+		Expect(tc.LoadImageToKindCluster()).To(Succeed())
+		Expect(tc.LoadImageToKindClusterWithName("quay.io/operator-framework/scorecard-test:dev")).To(Succeed())
+		Expect(tc.LoadImageToKindClusterWithName("quay.io/operator-framework/custom-scorecard-tests:dev")).To(Succeed())
```

Comment:
WDYT about add a func in the testutils LoadScorecardImages for example? It is used in many places.

Reply:
It certainly could. I could go either way.
1) If we leave it as is, there would be less indirection in the test code, which would make it easier to comprehend and debug, but if we change the scorecard image name (though we probably can't until 2.x), then we'd have to change this string in multiple places.
2) If we introduce a constant in a separate package, we decrease duplication and centralize the definition of the scorecard image names used in the e2e tests, but we add indirection and make the code slightly more difficult to directly read and understand.

In general, I would usually choose option 2, but I'm leaning toward 1 in this case for two reasons: the image names are _extremely_ unlikely to change for the duration of 1.x and it's nice for tests to have as little indirection as possible so that it is clear what the expectations of the test are.

But I'm good either way. If there's another vote for using constants, I'll switch it over.

Reply:
A 3. option would be a func such as LoadScorercardImages which would centralize all. Then, this func would try to load all images and not panic when the image X is not found to address the ansible/helm scenarios where is not required both.